src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -339,6 +340,16 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
     },
   ]
 
+  const hasAvatarUploadFailed =
```

Comment:
I think we could create some function, for all the assets(thumbnail, cover, avatar) to not repeat these lines over and over.